src/software/ai/evaluation/detect_successful_kick.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include "software/geom/vector.h"
+#include "software/world/ball.h"
+
+/**
+ * Software approximation that finds if a ball has been kicked, regardless of whether the
+ * kick was a pass, shot, or chip.
+ *
+ * @param ball The ball that can be kicked
+ * @param kick_direction The direction that we expect the ball to be kicked towards
+ *
+ * @return True if ball was kicked in the approximate direction we expect, false otherwise
+ */
+bool successfulKickDetected(const Ball& ball, const Vector& kick_direction);
```

Comment:
A couple comments here: 
1. `Vector` also represents a length, which is unused, so `kick_direction` should be an `Angle`
2. I think `min_kick_speed` should be a parameter that is default initialized to `0.5`, so that it can be overridden for very slow kicks.

Reply:
Thoughts on renaming to `hasBallBeenKicked`? Don't feel strongly about this so its fine to leave as is as well.

Reply:
changed to `Angle`, made `min_kick_speed` a parameter, and changed method name to `hasBallBeenKicked`